plugins/services/src/js/components/forms/GeneralServiceFormSection.js

```diff
@@ -240,23 +240,36 @@ class GeneralServiceFormSection extends Component {
   }
 
   getPlacementConstraintsFields(data = []) {
-    const errors = this.props.errors || [];
-    const showParameterLabel = data.some((constraint) => {
-      return ![GROUP_BY, UNIQUE].includes(constraint.operator);
+    const constraintsErrors = findNestedPropertyInObject(
+      this.props.errors,
+      'constraints'
+    );
+    const showValueLabel = data.some((constraint) => {
```

Comment:
Do we actually need `showValueLabel` and `showRequiredLabel` looks they always have the same values.